applications/openshift/api-server/api_server_api_priority_flowschema_catch_all/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Make sure there exists a catch-all FlowSchema object for API Priority and Fairness'
+
+description: |-
+    To make sure that all requests are controlled by the API Priority
+    and Fairness feature, make sure that there exists a catch-all rule.
+
+rationale: |-
+    Using <tt>APIPriorityAndFairness</tt> feature provides a fine-grained way
+    to control the behaviour of the Kubernetes API server in an overload
+    situation. The well-known FlowSchema <tt>catch-all</tt> should be available
+    to make sure that every request gets some kind of classification. By default,
+    the <tt>catch-all</tt> priority level only allows one concurrency share and
+    does not queue requests
```

Comment:
That's nice, but what is the security rationale?

Reply:
Fixed locally, will mark as resolved when I push new patches.